gating/scripts/vars.sh

```diff
@@ -13,6 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# The scripts expects the git clone to be at /opt/rpc-octavia, so we link
+# the current folder there.
+if [[ "${PWD}" != "/opt/rpc-octavia" ]]; then
+  ln -sfn ${PWD} /opt/rpc-octavia
+fi
+
```

Comment:
gating/scripts/pre.sh is running `source /opt/rpc-octavia/gating/scripts/vars.sh`, so perhaps this change needs to go into gating/scripts/pre.sh instead?